unison-cli/src/Unison/Cli/Share/Projects.hs

```diff
@@ -256,10 +256,9 @@ servantClientToCli action = do
   let clientEnv :: ClientEnv
       clientEnv =
         (mkClientEnv httpManager hardCodedBaseUrl)
-          { Servant.makeClientRequest = \url request ->
-              (Servant.defaultMakeClientRequest url request)
-                { Http.Client.responseTimeout = Http.Client.responseTimeoutMicro (60 * 1000 * 1000 {- 60s -})
-                }
+          { Servant.makeClientRequest = \url request -> do
```

Comment:
`makeClientRequest` in `servant-client-0.20` now returns `IO Request` https://hackage.haskell.org/package/servant-client-0.20/docs/Servant-Client.html#v:defaultMakeClientRequest